src/main/java/org/jenkinsci/plugins/credentialsbinding/impl/FileBinding.java

```diff
@@ -97,6 +98,7 @@ protected void copy(FilePath secret, FileCredentials credentials) throws IOExcep
         secret.copyFrom(credentials.getContent());
     }
 
+    @Symbol("file")
```

Comment:
Ah, okay I didn't realize the scoping role was per-extension.  But I still think calling it 'file' is less helpful than fileCredential (which is what it is, as opposed to things which read a file directly).